### PR TITLE
add the host name for lib-orange-alma-qa

### PR DESCRIPTION
**Pull request by kayiwa**
set the variables for the new VM and add them to vault

### CODE REVIEW

File: group_vars/orangelight/alma_qa.yml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+ol_feedback_to: "{{ vault_ol_feedback_to }}"
+ol_feedback_cc: "{{ vault_ol_feedback_cc }}"
+ol_newrelic_key: ""
+ol_newrelic_app_name: ""
+ol_honeybadger_key: ""
+ol_rabbit_user: "{{ vault_figgy_alma_qa_rabbit_user }}"
+ol_rabbit_password: "{{ vault_figgy_alma_qa_rabbit_password }}"
+ol_rabbit_host: "figgy-staging1.princeton.edu"
+ol_rabbit_server: "amqp://{{ ol_rabbit_user }}:{{ ol_rabbit_password }}@{{ ol_rabbit_host }}:5672"
+voyager_ub_id: "{{ vault_voyager_ub_id }}"
```
Comment (christinach) [inline group_vars/orangelight/alma_qa.yml:36]: for the qa-server we will have alma variables. I think we need to remove the voyager ones.

File: group_vars/orangelight/alma_qa.yml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  - libpq-dev
+  - pkg-config
+  - libtool
+  - autoconf
+passenger_ruby: "/usr/bin/ruby2.6"
+ruby_version_override: "ruby2.6"
+bundler_version: "2.1.4"
+postgresql_is_local: false
+postgres_host: "lib-postgres3.princeton.edu"
+postgres_version: 10
```
Comment (christinach) [inline group_vars/orangelight/alma_qa.yml:19]: lib-postgres-staging1.princeton.edu

File: group_vars/orangelight/alma_qa.yml [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    value: "{{ ol_host_name }}"
+  - name: APPLICATION_HOST_PROTOCOL
+    value: "{{ application_host_protocol }}"
+sneakers_workers: EventHandler
+sneakers_worker_name: orangelight-sneakers
+ol_qa_solr_url: "http://lib-solr8-staging.princeton.edu:8983/solr/catalog-qa"
+datadog_api_key: "{{ vault_datadog_key }}"
+datadog_config:
+  tags: "application:orangelight, environment:qa, type:webserver"
+  apm_enabled: "true"
```
Comment (christinach) [inline group_vars/orangelight/alma_qa.yml:93]: environment: alma-qa

File: group_vars/orangelight/alma_qa.yml [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+  apm_config:
+    analyzed_spans:
+      orangelight|rack.request: 1
+datadog_checks:
+  ruby:
+    init_config:
+    logs:
+      - type: file
+        path: /opt/rails_app/current/log/qa.log
+        service: orangelight
```
Comment (christinach) [inline group_vars/orangelight/alma_qa.yml:106]: path: /opt/rails_app/current/log/alma-qa.log

File: group_vars/orangelight/alma_qa.yml (hunk 1 above)
Comment (christinach) [reply group_vars/orangelight/alma_qa.yml:36]: The alma keys have been added in the lastpass

File: group_vars/orangelight/alma_qa.yml (hunk 4 above)
Comment (kayiwa) [reply group_vars/orangelight/alma_qa.yml:106]: done!

File: group_vars/orangelight/alma_qa.yml (hunk 3 above)
Comment (kayiwa) [reply group_vars/orangelight/alma_qa.yml:93]: done!

File: group_vars/orangelight/alma_qa.yml [hunk 5]
```diff
@@ -0,0 +1,120 @@
+---
+passenger_server_name: "catalog-alma-qa.*"
+passenger_app_root: "/opt/orangelight/current/public"
+passenger_app_env: "alma-qa"
+passenger_extra_config: '{{ lookup("file", "roles/pulibrary.orangelight/templates/nginx_extra_config")  }}'
```
Comment (christinach) [inline group_vars/orangelight/alma_qa.yml:5]: passenger_app_env: alma_qa

**Pull request closed without merging**